lib/Conversion/TorchToStablehlo/Pooling.cpp

```diff
@@ -373,6 +373,164 @@ LogicalResult ConvertAtenOp<AtenMaxPool2dWithIndicesOp>::matchAndRewrite(
   return success();
 }
 
+
+// AtenAvgPool1dOp
+template <>
+LogicalResult ConvertAtenOp<AtenAvgPool1dOp>::matchAndRewrite(
```

Comment:
Same comment as above. Common logic should be identified and refactored to avoid repetition.

Reply:
> Same comment as above. Common logic should be identified and refactored to avoid repetition.

Thank you! Have already merged this two Conversion by create a new template